packages/block-editor/src/components/inner-blocks/README.md

```diff
@@ -107,7 +107,7 @@ Template locking allows locking the `InnerBlocks` area for the current template.
 
 - `all` — prevents all operations. It is not possible to insert new blocks. Move existing blocks or delete them.
```

Comment:
Could also add quotation marks around the text to make it clearer that the first two options are strings.

```suggestion
- `'all'` — prevents all operations. It is not possible to insert new blocks. Move existing blocks or delete them.
```